lib/brakeman/commandline.rb

```diff
@@ -145,6 +145,13 @@ def regular_report options
           quit Brakeman::Errors_Found_Exit_Code
         end
 
+        if tracker.options[:ensure_no_obsolete_ignore_entries] && tracker.unused_fingerprints.any?
+          warn '[Error] The following obsolete ignore entries were found:'
```

Comment:
Obsolete ignore entries are already included in the output, so this shouldn't be necessary.

Reply:
Yep, makes sense. Fixed.